src/main/webapp/dosave.jsp

```diff
@@ -149,19 +150,34 @@
                     if (changed) {
                         String sql = "UPDATE " + zieltabelle + " SET ";
                         boolean sqlValid = false;
+
                         for (int i = 0; i < zielattributArray.length; i++) {
-                            if (request.getParameter(combinedFeldnamenArray[i]) != null && request.getParameter("Genauigkeit" + combinedFeldnamenArray[i]) != null) {
-                                if (i > 0) {
-                                    sql += ", ";
-                                }
-                                sql += zielattributArray[i] + "=\"" + ((!request.getParameter(combinedFeldnamenArray[i]).equals("")) ? request.getParameter(combinedFeldnamenArray[i]) : "0") + "\", ";
-                                sql += "Genauigkeit" + zielattributArray[i] + "=\"" + ((!request.getParameter("Genauigkeit" + combinedFeldnamenArray[i]).equals("")) ? request.getParameter("Genauigkeit" + combinedFeldnamenArray[i]) : "0") + "\"";
-                                sqlValid = true;
+                            String fieldValue = request.getParameter(combinedFeldnamenArray[i]);
+                            String genauigkeitValue = request.getParameter("Genauigkeit" + combinedFeldnamenArray[i]);
```

Comment:
As we discussed it yesterday, i think you were not yet able to test this in detail. Have you tried using this e.g. with certain date fields?